app/components/shared/notification_component.rb

```diff
@@ -1,17 +1,24 @@
 class Shared::NotificationComponent < ViewComponent::Base
-  def initialize(content:, style:, links: nil, dismiss: true, background: false, alert: "warning")
+  def initialize(content:, style:,
```

Comment:
I think we could benefit from a wholesale refactor of our approach to shared components like this (e.g passing custom classes and html_attributes consistently) now that we have more experience using them and their documentation has improved significantly. It's outside the scope of this ticket though so I'm happy for us to do this temporarily to achieve what you're trying to - and I'll make a ticket about what I want us to do eventually